data/templates/legacy-article.mst

```diff
@@ -11,13 +11,12 @@
     {{/title}}
     <section id="inside-content" class="section">
         <div>{{{body-html}}}</div>
-        <hr>
```

Comment:
I meant something else. What I meant is that before this patch we have:
```
<inside-content>
  <disclaimer>
</inside-content>
```

After this patch we have:

```
<inside-content>
  ...
</inside-content>
<disclaimer>
```

But, the CSS selectors still assume the previous structure [here](https://github.com/endlessm/eos-knowledge-lib/pull/1267/commits/cac136f4c92bc190d9eb830154bb58aa0f23d377#diff-db16cecbc2ecc49d958cd74c4b659d68R521), and I this is correct that selector should not be working.